src/ui/server/controllers/insurance/business/company-details/helpers/companies-house-search.helper.test.ts

```diff
@@ -134,11 +134,10 @@ describe('controllers/insurance/business/companies-details/helpers/companies-hou
 
     const response = await companiesHouseSearch(formBody);
 
-    const errorMessage = EXPORTER_BUSINESS_ERROR[COMPANY_HOUSE.INPUT].TECHNICAL_ISSUES;
-
     const expected = {
-      validationErrors: generateValidationErrors(COMPANY_HOUSE.INPUT, errorMessage, {}),
+      apiError: true,
       [COMPANY_HOUSE.INPUT]: formBody[COMPANY_HOUSE.INPUT],
+      validationErrors: {},
```

Comment:
Seems we don't have a test for the `validationErrors` being returned now?

Reply:
discussed, all other scenarios tested.  We redirect to another page when companies house is down